libcxx/test/std/input.output/string.streams/istringstream/istringstream.assign/copy.compile.pass.cpp

```diff
@@ -0,0 +1,25 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <sstream>
+
+// template <class charT, class traits = char_traits<charT>, class Allocator = allocator<charT> >
+// class basic_stringstream
+
+// basic_istringstream& operator=(const basic_istringstream&) = delete;
```

Comment:
I rather test this is a `types.compile.pass` this directly lets us test all copy/move properties in one place. I noticed there is already a file `test/std/input.output/string.streams/istringstream/types.pass.cpp` as a drive-by can you rename this file to `test/std/input.output/string.streams/istringstream/types.compile.pass.cpp` and remove the `main`? (Please check whether is applies to the other tests too.)